controlpanel/api/models/apps3bucket.py

```diff
@@ -24,9 +22,19 @@ class Meta:
         unique_together = ('app', 's3bucket')
         ordering = ('id',)
 
-    def aws_name(self):
+    @property
+    def iam_role_name(self):
```

Comment:
Good idea to keep this consistent across classes, as I was thinking in the `App` model we could potentially removed these from the model altogether, especially if they're just used in `cluster.*` classes. (but we can do later on if we want)